.github/workflows/pr-check-storybook.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build storybook on pull request
+
+on:
+  pull_request:
+    paths:
+      - 'package.json'
+      - 'yarn.lock'
+      - 'storybook/**'
```

Comment:
I would say that you want to check storybook if you modify the components as well
(let say I remove Button component or move it elsewhere I would hope that PR check fails)

Reply:
Oh yeah smart

Reply:
maybe just add an entry https://github.com/containers/podman-desktop/blob/main/.github/workflows/pr-check.yaml#L176-L221 if it's taking only few seconds

Reply:
We need to build the ui package first, the linter would not have the packages build I guess ?